scripts/konk-smoke-test.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash -x
+
+cd $(dirname $BASH_SOURCE)
+
+function cleanup {
+  k -n konk-test delete -f ../examples/konk.yaml
+}
+
+trap cleanup EXIT
+
+kubectl create ns konk-test || true
+kubectl -n konk-test apply -f ../examples/konk.yaml
+
+kubectl wait --timeout=1s --for=condition=Deployed -n konk-test konk runner-konk
+
+REASON=$(kubectl -n konk-test get konk runner-konk -o jsonpath="{range.status.conditions[?(@.type=='Deployed')]}{$@.type}={@.reason}{'\n'}{end}")
+
+if [[ $REASON != *"Successful" ]]; then
```

Comment:
As you mentioned before, this tests the wrong thing because you expect `runner-konk` to be a clean install, not an upgrade. If it's being upgraded, something went wrong with the cleanup stage.